### PR TITLE
Added basic react-router functionality

### PR DESCRIPTION
**Pull request by shergillmanav19**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,25 +15,36 @@ export function App(): JSX.Element | null {
...
+                                information.
+                              </Text>
+                            </VStack>
+                          </Center>
+                        )}
+                      </Flex>
+                    </Flex>
+                  </Box>
+                }
+              />
```
Comment (aomi) [inline src/App.tsx:45]: @keithradford should we move this code into a `pages` folder?

Comment (isaaccormack) [reply src/App.tsx:45]: agreed, or `routes` folder?

File: src/App.tsx [hunk 2]
```diff
@@ -15,25 +15,36 @@ export function App(): JSX.Element | null {
...
-                      </Text>
-                    </VStack>
-                  </Center>
-                )}
-              </Flex>
-            </Flex>
-          </Box>
+          <Router>
+            <Routes>
+              <Route path="/" />
```
Comment (isaaccormack) [inline src/App.tsx:20]: shouldn't the `/` route default to the app as we have now? As mentioned in the issue for this, theres no reason to make actual routes right now so I think only having this route, and the code in the route moved into a file in the `pages` folder (as discussed below) should suffice.

**Review by isaaccormack — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 2 above)
Comment (shergillmanav19) [reply src/App.tsx:20]: yep! I could just remove that route, but i think @aomi wanted a "/" route to a blank page and a "/calendar" route to the homepage we have right now

File: src/App.tsx (hunk 1 above)
Comment (keithradford) [reply src/App.tsx:45]: yeah, I think `routes` makes more sense given they aren't necessarily "pages". Either way I think a directory is justified

**Review by shergillmanav19 — COMMENTED** [review]
done

File: src/calendar.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+import { TermContext } from './app/context/TermContext';
+import { Header, Content, SidebarContainer } from './app/index';
+import { getCurrentTerm } from './app/shared/utils/terms';
+import { Term } from './fetchers';
+export function App(): JSX.Element | null {
+  const [term, setTerm] = useState(getCurrentTerm());
+  const [pid, setPid] = useState<string>();
+  const [subject, setSubject] = useState<string>('');
+  const [code, setCode] = useState<string>('');
```
Comment (aomi) [inline src/calendar.tsx:12]: This shouldn't have default empty strings.

File: src/calendar.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import { Box, Center, Flex, Heading, Text, VStack } from '@chakra-ui/react';
+import { useState } from 'react';
+
+import { TermContext } from './app/context/TermContext';
+import { Header, Content, SidebarContainer } from './app/index';
+import { getCurrentTerm } from './app/shared/utils/terms';
+import { Term } from './fetchers';
+export function App(): JSX.Element | null {
```
Comment (aomi) [inline src/calendar.tsx:8]: I would refactor this into `Calendar`

**Review by aomi — APPROVED** [review]
Once those two issues are resolved LGTM

**Review by isaaccormack — APPROVED** [review]

**Pull request merged**